projects/plugins/jetpack/_inc/class.jetpack-provision.php

```diff
@@ -36,12 +36,12 @@ function() use ( $url_arg, $named_args ) {
 
 		// If Jetpack is currently connected, and is not in Safe Mode already, kick off a sync of the current
 		// functions/callables so that we can test if this site is in IDC.
-		if ( Jetpack::is_connection_ready() && ! Jetpack::validate_sync_error_idc_option() && Actions::sync_allowed() ) {
+		if ( Jetpack::is_connection_ready() && ! Identity_Crisis::validate_sync_error_idc_option() && Actions::sync_allowed() ) {
```

Comment:
Same as above, would we need to import `Automattic\Jetpack\Identity_Crisis` here?

Reply:
:thumbsup: